fiftyone/core/odm/dataset.py

```diff
@@ -198,13 +205,19 @@ class ColorScheme(EmbeddedDocument):
                 document
             -   `valueColors` (optional): a list of dicts specifying colors to
                 use for individual values of this field
+        label_tags (None): an optional dict specifying custom colors for label tags
+            with the following keys:
+            -    `fieldColor` (optional): a color to assign to all label tags
+            -    `valueColors` (optional): a list of dicts specifying colors to
+            specific label tags
     """
 
     # strict=False lets this class ignore unknown fields from other versions
     meta = {"strict": False}
 
     color_pool = ListField(ColorField(), null=True)
     fields = ListField(DictField(), null=True)
+    label_tags = DictField(null=True)
```

Comment:
I think both of these fields should be typed. But it's not important for this PR